etc/telegraf.conf

```diff
@@ -2004,8 +2004,12 @@
 #   ## Setting 'use_lock' to true runs iptables with the "-w" option.
 #   ## Adjust your sudo settings appropriately if using this option ("iptables -wnvl")
 #   use_lock = false
+#   ## Define an alternate executable, such as "ip6tables". Default is "iptables".
+#   # binary = "ip6tables"
 #   ## defines the table to monitor:
 #   table = "filter"
+#   ## Define this to alter the measurement name. Default is "iptables".
+#   # measurement = ""
```

Comment:
We actually have built in support for this type of functionality on every [input plugin][]: `name_override = "foo"`

[input plugin]: https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#input-configuration